app/components/cases/parliament/description-panel.js

```diff
@@ -48,7 +48,7 @@ export default class CasesParliamentDescriptionPanelComponent extends Component
     );
 
     const publicationStatus = await this.store.queryOne('publication-status', {
-      'filter[publications][case][decisionmaking-flow][:id:]':
```

Comment:
Hmm, I think I was wrong here. `publications` is correct.
In these types of queries, we have to use the relation as defined in the model
in this case for the model of `publication-status`
`@hasMany('publication-flow', { inverse: 'status', async: true }) publications;`